proto/storage/product_usage.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto3";
+
+option go_package = "storage";
+option java_package = "io.stackrox.proto.storage";
+
+import "google/protobuf/timestamp.proto";
+import "gogoproto/gogo.proto";
+
+package storage;
+
+// SecuredUnits represents a record of an aggregated secured clusters usage
+// metrics. The metrics are aggregated periodically, and put into the database.
+message SecuredUnits {
+    // id is not used to retrieve data, but serves mostly for compatibility with
+    // the current implementation of the query generator.
+    string id                           = 1 [(gogoproto.moretags) = 'sql:"pk,type(uuid)"'];
+    // timestamp stores the moment at which the values of the metrics below are
+    // aggregated.
+    google.protobuf.Timestamp timestamp = 2;
+    // num_nodes is the maximum number of secured nodes, observed across all
+    // registered clusters during last aggregation period.
+    int64 num_nodes                     = 3;
```

Comment:
`max_num_nodes` or `max_node_count`

Reply:
This may be confused with the `max` service, which returns the maximum of those num_nodes.
I'd avoid having `max_max_num_nodes`.

Reply:
I am not convinced that renaming the var name to something that it actually represents will cause confusion. The comment clearly states it represents a max quantity.

Reply:
The comment is there to explain the meaning. The proper var name would be `max_num_nodes_aggregated_since_last_collection_period`, but we can just keep `num_nodes`, which represent somehow the recent aggregated usage.